runtime/interpreter/value_accountcapabilitycontroller.go

```diff
@@ -33,9 +33,9 @@ type AccountCapabilityControllerValue struct {
 	BorrowType   ReferenceStaticType
 	CapabilityID UInt64Value
 
-	// tag is locally cached result of GetTag, and not stored.
-	// It is populated when the field `tag` is read.
-	tag *StringValue
+	// Tag is locally cached result of GetTag, and not stored.
+	// It is populated when the field `Tag` is read.
```

Comment:
```suggestion
	// It is populated when the field `tag` is read.
```

Reply:
(see above commment)

Reply:
The comment/change suggestion here is unrelated to the comment above: "the field `tag`" here refers to the Cadence field, not the Go struct field. Same below